pallets/permissions/src/benchmarking.rs

```diff
@@ -0,0 +1,44 @@
+use crate::*;
+
+use polymesh_primitives::{DispatchableName, PalletName};
+
+use frame_benchmarking::benchmarks;
+use frame_support::ensure;
+use sp_std::{iter, prelude::*};
+
+const MAX_PALLET_NAME: u32 = 512;
+const MAX_DISPATCHABLE_NAME: u32 = 1024;
```

Comment:
very minor nitpick could you please add the `LENGTH` suffix to the variable name so it will be easy to understand.